optimum/intel/openvino/configuration.py

```diff
@@ -243,6 +255,11 @@ def post_init(self):
         if self.tokenizer is not None and not isinstance(self.tokenizer, str):
             raise ValueError(f"Tokenizer is expected to be a string, but found {self.tokenizer}")
 
+        if self.quant_method == QuantizationMethod.AWQ:
+            self.quant_method = OVQuantizationMethod.DEFAULT
+            self.awq = True
+            logger.warning('Using quant_method=\"AWQ\" is deprecated. Please use awq=True instead in the future.')
```

Comment:
I decided to change API for applying AWQ via source code, i.e. when we create `OVWeightQuantizationConfig`. 

Despite the fact there is an `QuantizationMethod.AWQ` enum, this approach does not quite fit the whole API of NNCF weight compression because AWQ is just another method, alike to Scale Estimation that can additionally be applied along a regular weight compression, and not some specific standalone method like hybrid quantization for example.

I've added a backward compatible logic to still apply AWQ if it was provided via previous API.

During export via optimum-cli it is already like this, e.g. there is an `--awq` argument.

Reply:
cc @AlexKoff88